Source/dom/WrappedObject.js

```diff
@@ -102,6 +102,26 @@ export class WrappedObject {
     return /Immutable/.test(String(this.sketchObject.className()))
   }
 
+  jsonArchive() {
```

Comment:
I’d rather go with `getSketchJSON`. “Archive” is not really explicit if you don’t know the underlying native API.